### ISSUE TITLE
PatientRepository.findAll() called twice in initial load.

### ISSUE THREAD

**Issue by akshay-ap** [HospitalRun/hospitalrun-frontend#2043]
## 🐛 Bug Report

`ViewPatients.tsx` calls `dispatch(searchPatients(debouncedSearchText))` and `dispatch(fetchPatients())` in initial load. Both of these functions call `PatientRepository.findAll()`.
So, patients list is fetched twice initially which is redundant.

## To Reproduce

Navigate to: `<host:port>/patients`

## Expected behavior

`PatientRepository.findAll()` called only once in initial load.

## Your Environment

- *node version*: 14
- *fastify version*: >=1.0.0
- *os*: Windows

This needs to be fixed with or before #2026.

**Comment by marcosvega91** [comment]
Hi @akshay-ap I reproduce the issue.
It doesn't make sense to have `dispatch(fetchPatients())` after #1970 
I think that in this case you can open a new PR and fix the issue otherwise I can do.

**Comment by akshay-ap** [comment]
I have the fix ready. I can raise the PR if you don't mind.

**Comment by marcosvega91** [comment]
Go forward 👯
